docs/source/quickstart/index.rst

```diff
@@ -16,11 +16,33 @@ See also the readme in the GitHub repository.
 Install Cloud Custodian
 -----------------------
 
+Cloud Custodian is a Python application and supports Python 2 and 3 on Linux and 
+Windows.
+
+We recommend using Python 3.6 or higher.
+
+Linux and Mac OS
++++++++++++++++++++++++++++
+
 To install Cloud Custodian, just run::
 
   $ python3 -m venv custodian
   $ source custodian/bin/activate
-  (custodian) $ pip install c7n
+  (custodian) $ pip install c7n       #Install AWS package
+  (custodian) $ pip install c7n_azure #Install Azure package
+  (custodian) $ pip install c7n_gcp   #Install GCP Package
+
```

Comment:
Will defer to @kapilt on this change.  It actually might be a pretty clear way to show users the different packages.  Technically `c7n` is the core package required for the others, but it is a dependency so it'll get installed automatically and it is OK for users to think about it this way.

@kapilt I assume once we have a chance to document/finish the install for it we might also recommend Cask/Docker as an option in the quickstart

Reply:
yeah.. I'd like to move to cask in the quick start. I've been seeing folks get into upgrade problems as well, that it would address (or via pip install -U) between dependencies.

Reply:
We'll add cask docs as a separate PR..